package.json

```diff
@@ -94,7 +92,6 @@
     "eslint-plugin-import": "2.31.0",
     "eslint-plugin-jest": "28.10.0",
     "eslint-plugin-jsdoc": "50.6.1",
-    "eslint-plugin-prettier": "5.2.1",
```

Comment:
Just to confirm, was this and references to its associated config removed because we have separate linting (both NPM script and `lint-staged` running `prettier`)?

Reply:
Yeah I followed the suggestion on this page:
https://typescript-eslint.io/users/what-about-formatting